### PR TITLE
Add motion submitter model.

### PR DESCRIPTION
**Pull request by normanjaeckel**
Draft version of motion_submitter.

Update motion_submitter model, use indirect meeting_id.

Draft version of the motion_submitter create action and tests.

### CODE REVIEW

**Comment by normanjaeckel** [comment]
@reiterl See my commit here: https://github.com/normanjaeckel/openslides-backend/tree/dev_motion_submitter You can fetch this into your branch and the tests should be fixed (for whatever reason I am not able to push into your branch).

File: openslides_backend/action/motion_submitter/create.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from typing import Any, Dict
+
+from ...models.motion_submitter import MotionSubmitter
```
Comment (normanjaeckel) [inline openslides_backend/action/motion_submitter/create.py:3]: Please use new models.py

File: openslides_backend/action/motion_submitter/delete.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from ...models.motion_submitter import MotionSubmitter
```
Comment (normanjaeckel) [inline openslides_backend/action/motion_submitter/delete.py:1]: Please use new models.py

File: openslides_backend/models/motion_submitter.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from ..shared.patterns import Collection
```
Comment (normanjaeckel) [inline openslides_backend/models/motion_submitter.py:1]: Please remove this file.

File: openslides_backend/action/motion_submitter/create.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/motion_submitter/create.py]: Done.

File: openslides_backend/action/motion_submitter/delete.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/motion_submitter/delete.py]: Done.

File: openslides_backend/models/motion_submitter.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/models/motion_submitter.py]: Done.

**Comment by reiterl** [comment]
I've added your patch to my branch, updated it and pushed it. 
The tests pass.

**Review by jsangmeister — APPROVED** [review]

File: openslides_backend/action/motion_submitter/create.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        check if motion and user have the same meeting_id
+        """
+        motion_meeting_id = self.database.get(
+            FullQualifiedId(Collection("motion"), instance["motion_id"]), ["meeting_id"]
+        ).get("meeting_id")
+        user_meeting_id = self.database.get(
```
Comment (jsangmeister) [inline openslides_backend/action/motion_submitter/create.py:30]: Just an FYI: user don't generally belong to a meeting (only if they are temporary), so don't assume that a user has a meeting_id. I seem to have missed this when merging this.

**Pull request merged**